db/change_cache.go

```diff
@@ -710,9 +754,11 @@ func (c *changeCache) _addToCache(change *LogEntry) base.Set {
 	// updatedChannels tracks the set of channels that should be notified of the change.  This includes
 	// the change's active channels, as well as any channel removals for the active revision.
 	updatedChannels := c.channelCache.AddToCache(change)
-	base.Debugf(base.KeyDCP, " #%d ==> channels %v", change.Sequence, base.UD(updatedChannels))
+	if base.LogDebugEnabled(base.KeyDCP) {
+		base.Debugf(base.KeyDCP, " #%d ==> channels %v", change.Sequence, base.UD(updatedChannels))
```

Comment:
> Avoid expensive base.UD on Set unless debug enabled

Not related to this PR, but it's given me an idea of how we can improve this across the board down in the logging/redaction levels by tweaking the way the redaction works. I'll file a Jira issue to track.